cfgov/v1/forms.py

```diff
@@ -126,7 +126,7 @@ def __init__(self, *args, **kwargs):
         self.hostname = kwargs.pop('hostname')
         self.base_query = kwargs.pop('base_query')
         super(FilterableListForm, self).__init__(*args, **kwargs)
-        page_ids = CFGOVPage.objects.live_shared(self.hostname).values_list('id', flat=True)
+        page_ids = self.base_query.live_shared(self.hostname).values_list('id', flat=True)
```

Comment:
Oh, yeah, I think that needs to be cleaned up. At some point it was defined in that function but now it's always passed in.